buildSrc/src/main/groovy/com/uber/okbuck/printable/PostProcessClassessCommands.groovy

```diff
@@ -23,10 +25,13 @@ public class PostProcessClassessCommands implements Printable {
     @Override
     public void print(PrintStream printer) {
         if (!mPostprocessClassesCommands.isEmpty()) {
-            String deps = "\$(JARS=(`find ${mGenDir} ! -name \"*-abi.jar\" ! -name \"*dex.dex.jar\" -name \"*.jar\"`); CLASSPATH=${mBootClasspath}; IFS=:; MERGED=( \"\${JARS[@]}\" \"\${CLASSPATH[@]}\" ); echo \"\${MERGED[*]}\")"
+            String deps = "\$(JARS=(`find ${mGenDir} ! -name \"*-abi.jar\" ! -name \"*dex.dex.jar\" -name \"*.jar\"`); " +
+                    "CLASSPATH=${mBootClasspath}; IFS=:; MERGED=( \"\${JARS[@]}\" \"\${CLASSPATH[@]}\" ); " +
+                    "echo \"\${MERGED[*]}\")"
+            String commandClassPath = mPostProcessDeps.join(":")
             printer.println("\tpostprocess_classes_commands = [")
             mPostprocessClassesCommands.each {
-                String command -> printer.println("\t\t'DEPS=${deps} ${command}',")
+                String command -> printer.println("\t\t'DEPS=${deps} COMMAND=${commandClassPath} ${command}',")
```

Comment:
I thought we agreed on putting the dependencies supplied in postprocess config as available as inline-var. This is basically what I'm doing there.